From: Mauro Carvalho Chehab <mchehab@redhat.com>
Subject: i7core_edac: remove static counter for max sockets
References: fate#311968
Patch-Mainline: yes
Git-commit: 0f062792b48dc8389fb18cbfb9318625886644c7


Signed-off-by: Thomas Renninger <trenn@suse.de>

The number of sockets is now fully dynamic. Get rid of this obsolete
var.

Signed-off-by: Mauro Carvalho Chehab <mchehab@redhat.com>

---
 drivers/edac/i7core_edac.c |    1 -
 1 file changed, 1 deletion(-)

Index: linux-2.6.32-SLE11-SP1/drivers/edac/i7core_edac.c
===================================================================
--- linux-2.6.32-SLE11-SP1.orig/drivers/edac/i7core_edac.c
+++ linux-2.6.32-SLE11-SP1/drivers/edac/i7core_edac.c
@@ -248,7 +248,6 @@ struct i7core_pvt {
 /* Static vars */
 static LIST_HEAD(i7core_edac_list);
 static DEFINE_MUTEX(i7core_edac_lock);
-static u8 max_num_sockets;
 
 #define PCI_DESCR(device, function, device_id)	\
 	.dev = (device),			\
